src/icons/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as LogoIcon } from "./logo-icon";
```

Comment:
Instead of using this format you can use `export LogoIcon from './logo-icon';`

Reply:
For some reasons this syntax does not work. Hence I used `export { default as LogoIcon } from "./logo-icon";`